File: Assignment 5/Activity 2.js
function main() {
    // This program converts the unit of Years into months,days,hours, minutes and seconds.
    var years;
    var months;
    var days;
    var hours;
    var seconds;
    
    years = getyears();
    months = calculatemonths(years);
    days = calculatedays(years);
    hours = calculatehours(years);
    seconds = calculateseconds(years);
    displayResult(months, days, hours, seconds);
}

function calculatedays(years) {
    var days;
    
    days = years * 365;
    
    return days;
}

function calculatehours(years) {
    var hours;
    
    hours = years * 8760;
    
    return hours;
}

function calculatemonths(years) {
    var months;
    
    months = years * 12;
    
    return months;
}

function calculateseconds(years) {
    var seconds;
    
    seconds = years * 31536000;
    
    return seconds;
}

function displayResult(months, days, hours, seconds) {
    window.alert(months.ToString() + " months " + days + " days " + hours + " hours " + seconds + " seconds ");
}

function getyears() {
    var years;
    
    window.alert("How old are you?");
    years = window.prompt('Enter a value for years');
    
    return years;
}

File: Assignment 5/Activity 2.py
def calculatedays(years):
    days = years * 365
    
    return days

def calculatehours(years):
    hours = years * 8760
    
    return hours

def calculatemonths(years):
    months = years * 12
    
    return months

def calculateseconds(years):
    seconds = years * 31536000
    
    return seconds

def displayResult(years, months, days, hours, seconds):
    print(str(years) + " years " + str(months) + " months " + str(days) + " days " + str(hours) + " hours " + str(seconds) + " seconds ")

def getyears():
    print("How old are you?")
    years = float(input())
    
    return years

# Main
# This program converts the unit of Years into months, days, hours, and seconds.
years = getyears()
months = calculatemonths(years)
days = calculatedays(years)
hours = calculatehours(years)
seconds = calculateseconds(years)
displayResult(years, months, days, hours, seconds)
